site/docs/5_references/1_nifi_cluster/1_nifi_cluster.md

```diff
@@ -193,6 +194,15 @@ spec:
 | searchBase   | string  | base DN for searching for users (i.e. CN=Users,DC=example,DC=com).                                                                        | No       | ""      |
 | searchFilter | string  | Filter for searching for users against the 'User Search Base'. (i.e. sAMAccountName={0}). The user specified name is inserted into '{0}'. | No       | ""      |
 
+## SingleUserConfiguration
+
+| Field             | Type    | Description                                                                                                                               | Required | Default |
+| ----------------- | ---------------- | ----------------------------------------------------------------------------------------------------------------------------------------- | -------- | ------- |
+| enabled           | boolean          | if set to true, we will enable single-user usage into nifi.properties configuration.                                                                     | No       | false   |
+| authorizerEnabled | boolean          | if set to true, we will use the single-user-authorizer instead of the managed-authorizer.                                                                | No       | false   |
+| secretRef         | [*SecretReference](../4_nifi_parameter_context#secretreference)   | the reference to the username and password. If not configured, will use auto generated username and password that will appear once in the logs at startup.      | No       | nil     |
+| secretKeys         | *SecretKeys   | the keys of the username and password.      | No       | {username= "username", password="password"}     |
```

Comment:
A documentation section should be add for the SecretKeys doc to describe it. Like it's done for the SecretReference.
Directly in the 1_nifi_cluster.md file.

Reply:
Section added and link to the new section made in singleuserconfiguration